packages/nimble-components/package.json

```diff
@@ -30,6 +30,8 @@
     "test-chrome:watch": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --watch-extensions js",
     "test-chrome": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --single-run",
     "test-firefox": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run",
+    "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run",
+    "test-webkit:debugger": "karma start karma.conf.js --browsers=Webkit",
     "test": "npm run test-chrome:verbose"
```

Comment:
We don't actually seem to be running firefox on the CI (not included here), does it also have existing failures?
If not maybe we should turn it on (might have been because we didn't have an npm managed version of firefox to run against before switching over to playwright)

Reply:
We do have existing test failures in Firefox unfortunately (different tests than WebKit).

Reply:
Also created work items for the test failures - #1074  and #1075